CruCentralCoast/Login/LoginManager.swift

```diff
@@ -48,6 +48,18 @@ class LoginManager: NSObject {
             completion?()
         }
     }
+    
+    func logout(sender viewController: UIViewController?) {
+        do {
+            try Auth.auth().signOut()
+            NotificationCenter.default.post(name: NSNotification.Name.UserDidLogout, object: nil)
```

Comment:
could you just do `.UserDidLogout`?

Reply:
Yep 🤦‍♂️
That was leftover from when I was making a new Notification Name.
I will fix this